<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactBook</class>
 <widget class="QDialog" name="ContactBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>è¯çµ¡ç°¿ç®¡ç†ç³»çµ±</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f5f5f5;
}

QTableWidget {
    border: 2px solid #d0d0d0;
    border-radius: 8px;
    background-color: white;
    gridline-color: #e0e0e0;
    font-size: 12pt;
}

QTableWidget::item {
    padding: 8px;
}

QTableWidget::item:selected {
    background-color: #4a90e2;
    color: white;
}

QHeaderView::section {
    background-color: #4a90e2;
    color: white;
    padding: 8px;
    border: 1px solid #3a7bc8;
    font-weight: bold;
    font-size: 12pt;
}

QLineEdit {
    border: 2px solid #d0d0d0;
    border-radius: 6px;
    padding: 8px;
    background-color: white;
    font-size: 11pt;
    min-height: 30px;
}

QLineEdit:focus {
    border: 2px solid #4a90e2;
}

QLabel {
    font-size: 11pt;
    font-weight: bold;
    color: #333333;
}

QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 10px 20px;
    font-size: 11pt;
    font-weight: bold;
    min-width: 80px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #357abd;
}

QPushButton:pressed {
    background-color: #2a5f96;
}

QPushButton#exitButton {
    background-color: #e74c3c;
}

QPushButton#exitButton:hover {
    background-color: #c0392b;
}

QPushButton#exitButton:pressed {
    background-color: #a93226;
}

QPushButton#insertButton {
    background-color: #27ae60;
}

QPushButton#insertButton:hover {
    background-color: #229954;
}

QPushButton#insertButton:pressed {
    background-color: #1e8449;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>ğŸ“‡ å­¸ç”Ÿè¯çµ¡ç°¿ç®¡ç†ç³»çµ±</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18pt; font-weight: bold; color: #2c3e50; padding: 10px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableW">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>æ–°å¢è¯çµ¡äººè³‡è¨Š</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font-size: 12pt;
    font-weight: bold;
    border: 2px solid #d0d0d0;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 10px;
    background-color: white;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 10px;
    color: #2c3e50;
}</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ğŸ“ å­¸è™Ÿï¼š</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ğŸ« ç­ç´šï¼š</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ğŸ‘¤ å§“åï¼š</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ğŸ“ é›»è©±ï¼š</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="insertButton">
       <property name="text">
        <string>â• æ–°å¢</string>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>ğŸ“¥ åŒ¯å…¥</string>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>ğŸ“¤ åŒ¯å‡º</string>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>ğŸ—‘ï¸ åˆªé™¤</string>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#deleteButton {
    background-color: #e67e22;
}

QPushButton#deleteButton:hover {
    background-color: #d35400;
}

QPushButton#deleteButton:pressed {
    background-color: #ba4a00;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>âŒ çµæŸ</string>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
